doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -939,7 +939,60 @@ Response:
             }
         }
 
-### Update or append entity attributes [POST /v2/entities/{entityId}{?type,options}]
+### Retrieve entity attributes [GET /v2/entities/{entityId}/attrs{?type,attrs,options}]
+
+Similar to retreiving the whole entity, this operation omits id an type fields.
```

Comment:
typo: an => and.
Perhaps `id` and `type` should be 'highlighted' somehow ... easier to read that way.

Also, I would add a 'however' after the comma, and start with "This request is similar ..."